modal/runner.py

```diff
@@ -195,7 +198,9 @@ async def _publish_app(
         function_ids=running_app.function_ids,
         class_ids=running_app.class_ids,
         definition_ids=definition_ids,
+        commit_info=api_pb2.CommitInfo(**commit_info) if commit_info else None,
```

Comment:
Actually seeing how you're using it here; can probably just use the `api_pb2.CommitInfo` as the primary data structure since it's all internal.